app/domain/export/tabular/people/beitragskategorie_wechsel_scope.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2025, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module Export::Tabular::People
+  class BeitragskategorieWechselScope < MutatedRolesScope
+    def roles
+      roles_scope.where(id: relevant_person_ids)
```

Comment:
Ist das nicht doppelt gemoppelt? Da werden die ids geladen, in denselben Scope verpackt und dann im Tabular wiederum die ids für ein Subquery extrahiert. Die korrekten ids (`person_id`) könnten direkt im Tabular verwendet werden.